src/rdkafka_broker.c

```diff
@@ -3649,6 +3652,7 @@ static int rd_kafka_toppar_producer_serve(rd_kafka_broker_t *rkb,
                                           rd_ts_t now,
                                           rd_ts_t *next_wakeup,
                                           rd_bool_t do_timeout_scan,
+                                          int *timeout_cnt,
```

Comment:
name this `timeout_cntp` (p indicating that it is a pointer, otherwise it is too easy to do `timeout_cnt++`)